CMakeLists.txt

```diff
@@ -94,6 +94,11 @@ cmake_dependent_option(FUZZ "Build for fuzzing. Enabling this will disable all o
 
 option(INSTALL_MAN "Install man pages." ON)
 
+set(EXTRA_CPPFLAGS "" CACHE STRING "Extra (Objective) C/C++ preprocessor flags.")
```

Comment:
If CMake does not have a notion of `CPPFLAGS` can we avoid it as well? Looks like sneaking an autotool-ism into CMake based build system. I peeked at the CI changes that engage `EXTRA_CPPFLAGS` and I think using `EXTRA_CXXFLAGS` instead should work, no?